doc/getting-started/rt_industry.rst

```diff
@@ -0,0 +1,270 @@
+   :name: Hardware Setup
+
+   +----------------------+-------------------+----------------------+-----------------------------------------------------------+
+   | Platform (Intel x86) | Product/kit name  | Hardware             | Descriptions                                              |
+   +======================+===================+======================+===========================================================+
+   | Kaby Lake            | NUC7i7DNH         | Processor            | - Intel |reg| Core |trade| i7-8650U CPU @ 1.90GHz         |
+   |                      |                   +----------------------+-----------------------------------------------------------+
+   |                      |                   | Graphics             | - UHD Graphics 620                                        |
+   |                      |                   |                      | - Two HDMI* 2.0a ports supporting 4K at 60 Hz             |
+   |                      |                   +----------------------+-----------------------------------------------------------+
+   |                      |                   | System memory        | - 8GiB SODIMM DDR4 2400 MHz                               |
+   |                      |                   +----------------------+-----------------------------------------------------------+
+   |                      |                   | Storage capabilities | - SATA: 1TB WDC WD10SPZX-22Z                              |
+   |                      |                   |                      | - NVMe: 256G Intel Corporation SSD Pro 7600p/760p/E 6100p |
+   +----------------------+-------------------+----------------------+-----------------------------------------------------------+
+
+How to set up ACRN Hypervisor for industry scenario
+***************************************************
+There are several ways to set up ACRN industry scenario environment. Two of them listed below
+are recommended:
+
+- :ref:`Using the pre-installed industry ACRN hypervisor <use pre-installed industry efi>`
+- :ref:`Using the ACRN industry out-of-the-box image <use industry ootb image>`
+
+.. _use pre-installed industry efi:
+
+Using the pre-installed industry ACRN hypervisor
+================================================
+Firstly we need to follow :ref:`ACRN quick setup guide <quick-setup-guide>` to set up
+ACRN Service VM. The industry hypervisor image is installed in ``/usr/lib/acrn/``
+directory once the Service VM boots. You may simply follow these steps to use
+``acrn.kbl-nuc-i7.industry.efi`` instead of the original SDC hypervisor:
+
+.. code-block:: none
+
+   $ sudo mount /dev/sda1 /mnt
+   $ sudo mv /mnt/EFI/acrn/acrn.efi /mnt/EFI/acrn/acrn.efi.bak
+   $ sudo cp /usr/lib/acrn/acrn.kbl-nuc-i7.industry.efi /mnt/EFI/acrn/acrn.efi
+   $ sync && umount /mnt
+   $ sudo reboot
+
+.. note:: Skip this section if you choose following the :ref:`Using the ACRN industry out-of-the-box image <use industry ootb image>`
```

Comment:
If this note refers to the section above, we'd better put it at the start of it. It's better to be warned you should skip something **before** you go through it rather than **after** :-)

Reply:
Done